MAUI/LinearGauge/animation.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Animation in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing animation of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Animation in .NET MAUI Linear Gauge (SfLinearGauge)
+
+All Linear Gauge elements such as axis along with ticks and labels, range, bar pointer, shape marker pointer and content marker pointer can be animated separately. 
```

Comment:
All Linear Gauge elements including the axis, ticks and labels, range, bar pointer, shape marker pointer, and content marker pointer, can be animated separately.